server/helpers/nestedDataProvider.js

```diff
@@ -33,12 +33,24 @@ const ratingsModelObj = {
   attributes: ['stars', 'userId']
 };
 
+const clapsModelObj = {
+  model: models.ArticleClap,
+  as: 'claps',
+  attributes: ['userId', 'claps'],
+};
+
+const commentsModelObj = {
+  model: models.Comment,
+  as: 'comments',
+};
+
 /**
  *
  *
  * @param {*} categoryQuery - category Id needed to query the database
  * @param {*} authorQuery - author name needed to query the DB
  * @param {*} tagQuery - tag name needed to query the DB
+ * @param {*} clapsQuery - tag name needed to query the DB
```

Comment:
this still has the description for `tagQuery`

Reply:
Done. Thanks for pointing it out.